modules/descargas/routes/descargas.ts

```diff
@@ -74,4 +75,91 @@ router.post('/constanciaPuco/:tipo?', (req: any, res, next) => {
     });
 });
 
+
+router.post('/internaciones/Csv', async (req, res, next) => {
+    const csv = require('fast-csv');
+    const fs = require('fs');
+    let ws = fs.createWriteStream('/tmp/internaciones.csv', { encoding: 'utf8' });
+
+    if (req.body.filtros && req.body.organizacion) {
+        try {
+            let data = await exportarInternacionesAnielo(req.body.filtros, req.body.organizacion);
+            csv
+                .write(data, {
+                    headers: true, transform: (row) => {
+                        return {
```

Comment:
Esta linea se pued resumir con 
```suggestion
                        return { ...row }
```

Ya que no se hace ninguna transformación extra.